platform-hub-web/src/app/identities/git_hub_identity.service.js

```diff
@@ -0,0 +1,18 @@
+export const gitHubIdentityService = function (windowPopupService, apiEndpoint, homeEndpoint) {
+  'ngInject';
+
+  const service = {};
+
+  service.popupFlow = popupFlow;
```

Comment:
I'm following a pattern here for service definitions: even if it just the one method exported it's (slightly) nicer when adding new methods in the future to declare things like this (not to mention consistent when glossing over files).